api/meta_protocol_proxy/v1alpha/meta_protocol_proxy.proto

```diff
@@ -42,11 +42,14 @@ message MetaProtocolProxy {
   // The codec which encodes and decodes the application protocol.
   Codec codec = 5;
 
+  // for idle downstream timer.
+  google.protobuf.Duration idle_timeout = 6;
+
   // A list of individual Layer-7 filters that make up the filter chain for requests made to
   // the meta protocol proxy. Order matters as the filters are processed sequentially as
   // request events happen. If no meta_protocol_filters are specified, a default router filter
   // (`aeraki.meta_protocol.filters.router`) is used.
-  repeated MetaProtocolFilter meta_protocol_filters = 6;
```

Comment:
This change is not backward compatible. Please keep the order of meta_protocol_filters.

idle_timeout can start form 11, so we have some room for future changes.

Reply:
ok.i will change the filed idle_timeout to 11.